### PR TITLE
Added template for command docs

### PR DESCRIPTION
**Pull request by apoorvyadav1111**
Hi team,

Added an empty .md file for command docs based on `SET` documentation. It is to be used by contributors for auditing and added documentation.

Thanks
Apoorv

### CODE REVIEW

**Comment by apoorvyadav1111** [comment]
Hi @AshwinKul28, Please review this template. All the contributors can use this template for standard documentation.

Thanks,
Apoorv

File: docs/command_docs_template.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+## Parameters
+<!-- add all parameters, small description, type and required -->
+
+| Parameter | Description                                                               | Type    | Required |
+|-----------|---------------------------------------------------------------------------|---------|----------|
+| `key`     | The name of the key to be set.                                            | String  | Yes      |
+|           |                                                                           |         |       |
+
+
+## Return values
```
Comment (AshwinKul28) [inline docs/command_docs_template.md:23]: | Condition                                       | Return Value                                       |
|-------------------------------------------------|----------------------------------------------------|
| If key is set successfully                      | "OK"                                               |
| If `NX` is used and key already exists          | nil                                                |
| If `XX` is used and key doesn't exist           | nil                                                |


Can we add this as an example for return values?

Comment (apoorvyadav1111) [reply docs/command_docs_template.md:23]: Yes, I ll add these.

File: docs/command_docs_template.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+The `SET` command in DiceDB is used to set the value of a key. If the key already holds a value, it is overwritten, regardless of its type. This is one of the most fundamental operations in DiceDB as it allows for both creating and updating key-value pairs.
+
+## Syntax
+
+```bash
+SET key value [EX seconds | PX milliseconds | EXAT unix-time-seconds | PXAT unix-time-milliseconds | KEEPTTL] [NX | XX]
+```
+
+## Parameters
+<!-- add all parameters, small description, type and required -->
```
Comment (AshwinKul28) [inline docs/command_docs_template.md:15]: | Parameter   | Description                                                              | Type     | Required |
|-------------|--------------------------------------------------------------------------|----------|----------|
| `key`       | The name of the key to be set.                                            | String   | Yes      |
| `value`     | The value to assign to the key.                                           | String   | Yes      |
| `EX`        | Set the key’s expiration time, in seconds.                                | Integer  | No       |
| `PX`        | Set the key’s expiration time, in milliseconds.                           | Integer  | No       |
| `EXAT`      | Set the expiration as an absolute UNIX timestamp in seconds.              | Integer  | No       |
| `PXAT`      | Set the expiration as an absolute UNIX timestamp in milliseconds.         | Integer  | No       |
| `KEEPTTL`   | Retain the key's existing TTL when updating its value.                    | Flag     | No       |
| `NX`        | Set the key only if it does not already exist.                            | Flag     | No       |
| `XX`        | Set the key only if it already exists.

Just a thought, shall we consider adding all supported parameters info in this sample doc to give idea to contributors to consider this as well?

**Comment by AshwinKul28** [comment]
Thanks a lot, @apoorvyadav1111 for this. This is a robust documentation example that can be followed throughout. Great! Please have a look on couple of thoughts.

File: docs/command_docs_template.md (hunk 2 above)
Comment (apoorvyadav1111) [reply docs/command_docs_template.md:15]: Yes, we can do that. I will add these along with others. Using comments to explain and guide what to modify in each section.

**Comment by apoorvyadav1111** [comment]
Hi @AshwinKul28 , updated the template with examples and more comments. Thanks for your suggestions.

**Comment by AshwinKul28** [comment]
Amazing thanks. one last thought @apoorvyadav1111 , This looks more like a perfect ideal example than a template, shall we have a separate file just for the template having all sections/subsections where users can pick and start adding their content and this in a separate file or top/bottom in template file as an ideal example? I feel the template should be more generalized. wdyt? I'm just thinking from general audience.

**Comment by apoorvyadav1111** [comment]
Hi @AshwinKul28 . This sounds even better what we have right now. So we can have a sample_doc file which can have all sections (notes, depreciation warnings) etc from different commands as an example. Additionally, We can have a template_doc which is just for structure with no comments and no text.

Please confirm my understanding about the requirements.

**Comment by apoorvyadav1111** [comment]
Hi @AshwinKul28 , added template and sample separately. Currently, all docs have some deviation from each other which should not be a time consuming task but will have multiple files being affected. Once this is approved, we can fix those headings. 
Thanks

**Comment by AshwinKul28** [comment]
Perfect, thanks for the changes @apoorvyadav1111 . LGTM

**Review by AshwinKul28 — APPROVED** [review]

**Pull request merged**